### ISSUE TITLE
Reseting a project does not eliminate imported images and generated CSV files

### ISSUE THREAD

**Issue by Miguel-Hermanns** [sdaps/sdaps#204]
When I reset a project, I see that the imported questionnaires (tif file) as well as the generated csv file remain in the project directory, so that importing a new set of questionnaires and generating a new csv leads to two tif files and two csv files (with an increasing counter in their names). 

Shouldn't the reseting of a project also clear these files?

**Comment by benzea** [comment]
Hmm, I wonder how to handle files outside/inside the project directory though (i.e. the `--no-copy` option for `add`).

Of course, it is trivial to just delete all `.tif` files yourself. But I do see how it would be convenient and that it is kind of unexpected.

**Comment by Miguel-Hermanns** [comment]
I would probably delete de CSV files as well, for consistency. Or?

**Comment by benzea** [comment]
Yeah ... deleting all of the file types we would generate likely makes sense.

There is a slight risk of us deleting files that belong to the LaTeX questionnaire document, as we do not record what has been copied in using `sdaps setup tex --add X`.

Maybe the ideal thing would be to either:
 1. Make `reset` ask for confirmation (and suggest making a copy first)
 2. Add a `setup existing` or so command. So, conceptually create a new empty project instead of resetting an existing one.
